AutomatedTesting/Gem/PythonTests/AWS/README.md

```diff
@@ -20,17 +25,19 @@
 3. Open a new CLI window at the engine root and set the following environment variables:
     * Windows
         ```
-            Set O3DE_AWS_PROJECT_NAME=AWSAUTO
-            Set O3DE_AWS_DEPLOY_REGION=us-east-1
-            Set ASSUME_ROLE_ARN=arn:aws:iam::{your_aws_account_id}:role/o3de-automation-tests
-            Set COMMIT_ID=HEAD
+            set O3DE_AWS_PROJECT_NAME=AWSAUTO
+            set O3DE_AWS_DEPLOY_REGION=us-east-1
+            set ASSUME_ROLE_ARN=arn:aws:iam::{your_aws_account_id}:role/o3de-automation-tests
+            set COMMIT_ID=HEAD
+            set CDK_VERSION=2.39.1

```

Comment:
I think we need to update the CDK version in these files as well to run the tests on Jenkins with CDK v2: 

- https://github.com/o3de/o3de/blob/development/scripts/build/Platform/Windows/build_config.json#L507
- https://github.com/o3de/o3de/blob/development/scripts/build/Platform/Linux/build_config.json#L381

Reply:
Good call, Will update those.